src/data.js

```diff
@@ -16477,6 +16477,40 @@ module.exports = [
       'JAMstack',
     ],
   },
+  {
+    name: 'Fortune Obi',
+    description:
+      'Web developer living in Canada, strangely obsessed with Typescript and React.',
+    url: 'https://www.obifortune.com/uses',
```

Comment:
Looks like this returns a 404 page right now

Reply:
Sorry about that! I was migrating my website and hadn't moved that page it yet.

It should be up now!